ci/compute-matrix.jq

```diff
@@ -14,6 +14,30 @@ def compute_arch($x):
 def matches($entry; $exclude):
   all($exclude | to_entries | .[]; $entry[.key] == .value);
 
+def compute_repo($x):
+  if
+    env.BUILD_TYPE == "pull-request"
+  then
+    "staging"
+  else
+    $x.IMAGE_REPO
+  end;
+
+def compute_tag_prefix($x):
+  if
+    env.BUILD_TYPE == "branch"
+  then
+    ""
+  else
+    $x.IMAGE_REPO + "-" + env.PR_NUM + "-"
+  end;
+
+def compute_image_name($x):
+  compute_repo($x) as $repo |
+  compute_tag_prefix($x) as $tag_prefix |
+  "rapidsai/" + $repo + ":" + $tag_prefix + "cuda" + $x.CUDA_VER + "-" + $x.LINUX_VER + "-" + "py" + $x.PYTHON_VER |
```

Comment:
```suggestion
  "rapidsai/" + $repo + ":" + $tag_prefix + "cuda" + $x.CUDA_VER + "-base-" + $x.LINUX_VER + "-" + "py" + $x.PYTHON_VER |
```

The existing `mambaforge-cuda` image names have `-base-` in the tag name: https://hub.docker.com/r/rapidsai/mambaforge-cuda/tags

Reply:
Good catch. Will fix